src/main/java/com/google/cloud/hadoop/io/bigquery/hive/HiveBigQueryConstants.java

```diff
@@ -21,10 +21,19 @@
   public static final String UNIQUE_JOB_KEY = "mapred.bq.unique.job.id";
   public static final String DEFAULT_BIGQUERY_DATASET_KEY = "bq.dataset";
   public static final String DEFAULT_BIGQUERY_TABLE_KEY = "bq.table";
+  public static final String PREDICATE_PUSHDOWN_COLUMNS = "ppd.columns";
```

Comment:
It's a table property name that program attaches to the table to track types that can be used for predicate push down.  Property name will be visible on each table definition. Clear name would be predicate.pushdown.columns but worried about length restriction.  Since its more for internal usage can we leave the name in short form ppd.columns?